python/py-json-tricks/Portfile

```diff
@@ -28,14 +28,24 @@ long_description    The pyjson-tricks package brings several pieces of \
                     Fraction, enums, compression, duplicate keys, ...
 
 checksums \
-    rmd160  9871b41beca7a12e96150a4f4605180dcda5b033 \
-    sha256  c18058ab327af08a311fc6a52745606a7834e65692d6b64f2767e47f7ada2b86 \
-    size    37521
+    rmd160  ecc7856a90794c741e949743bdad9c1d8c4d5973 \
+    sha256  04989867625dd1a25746f32b6b2006b6d3508e27c908053fcf564e3b71526adc \
+    size    41636
 
 if {${name} ne ${subport}} {
     livecheck.type          none
     depends_build-append    port:py${python.version}-setuptools
 
+    depends_test-append     port:py${python.version}-pytest \
+                            port:py${python.version}-numpy \
+                            port:py${python.version}-tz \
+                            port:py${python.version}-pandas \
+                            port:py${python.version}-numpy
```

Comment:
Please sort these alphabetically. Then it will be easy to spot duplicates such as py-numpy.